horovod/spark/lightning/estimator.py

```diff
@@ -106,8 +106,8 @@ class TorchEstimator(HorovodEstimator, TorchEstimatorParamsWritable,
                     Defaults to SparkBackend with `num_proc` worker processes. Cannot be specified
                     if `num_proc` is also provided.
         batch_size: Number of rows from the DataFrame per batch.
-        data_loader_class:  (Optional) Class of the custom data loader, if not set, lightning
-                            trainer will use PythonAsyncDataLoader as default.
+        data_module: (Optional) Lightning datamodule used for training and validadation, if not set,
+                    lightning trainer will use PetastormDataModule as default.
```

Comment:
Why change default to PetastormDataModule ? I thought  PythonAsyncDataLoader is faster?

Reply:
A typo. It is using `PythonAsyncDataLoader `

Reply:
To correct, PetastormDataModule  uses PythonAsyncDataLoader internally, but we don't have to expose this to users.